tests/queries/0_stateless/01497_now_support_timezone.sql

```diff
@@ -0,0 +1 @@
+SELECT toHour(now('Asia/Shanghai')) - toHour(now('UTC'));
```

Comment:
This query depends on the time when it is run.

Suppose now is 11 PM in UTC and 7 AM in Shanghai. The difference will be `-4`, not `8`.